utils/internal/testutils/tmpfs.go

```diff
@@ -0,0 +1,75 @@
+package testutils
```

Comment:
the convention for test utils is to create a subfolder with `test` suffix of the module of interest such as `filesystemtest` in `filesystem` and add the utilities to a file called testing.go. The test util should also take a t*testing.T argument and the first statement should be t.Helper()